### ISSUE TITLE
The color contrast of field and radio button interface components on the Edit Profile page is too low

### ISSUE THREAD

**Issue by rpkoller** [AzuraCast/AzuraCast#6680]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

#d64b1a4 (2023-10-20 0:01)

### What happened?

The color contrast for interface components like the field or radio buttons doesn't satisfy wcag 2.1 SC 1.4.11. 

![Screenshot 2023-10-20 at 03 45 52](https://github.com/AzuraCast/AzuraCast/assets/1665422/3e28c129-7753-4aba-8de0-8b9926cb703f)

`--bs-border-color` is `#dee2e6` that leads against a background of white to a color contrast of `1.3:1`. 1.4.11 requires at least a color contrast of `3:1`.

**Comment by rpkoller** [comment]
i've tested in the new rolling release and it is much much better. "technically speaking slightly under the required minimum 3:1 with 2.9:1 but that should be fine ;) but i completely forgot two details. with the applied changes on the loging it is apparent now that the checkbox outline should be updated as well. aaaand the main thing i forgot is the dark theme. there the color contrast for those interface components would need an update as well. but for dark mode there is a more serious problem in regards of SC1.4.11

![Screenshot 2023-10-21 at 13 08 54](https://github.com/AzuraCast/AzuraCast/assets/1665422/dbe7fef0-2148-4e30-aa40-38f1fe3a1ebe)

to clearly visually communicate the extents of the dialog modal. for the boxes on the my account page it is the same. you only see the blue header, the background of the box is the same as the background color of the page there is only a "dark"? border with the color of #1A1B23 that has a color contrast of 1:1 to the background of #22252F 

![Screenshot 2023-10-21 at 13 10 24](https://github.com/AzuraCast/AzuraCast/assets/1665422/9319ce7a-6957-41ff-b053-063762e6e112)

i usually don't need to check the color, my conditions with my eyes let me struggle with too low color contrast and i can easily spot challenging hard to read parts.
